### PR TITLE
Changed "technologies" and "tools" chip labels to singular chip labels (technology and tool) - #4647

### LINKED ISSUE #4647: Update chip labels from plural to singular
### Dependency
- [x] #4960

### Overview
Currently, chip label prefixes for the "Technologies" and "Tools" filter groups are pluralized even if only one filter item is listed under the label. This can be confusing to users, so we want to update the chip label prefixes to "Technology" and "Tool"

### Action Items

- [x] Check the current state of search chip labels under Applied Filters on the [Projects page](https://www.hackforla.org/projects/).
  - [x] Apply some of the filters to see how the current chips appear under Applied Filters.
- [x] Explore `assets/js/current-projects.js` and understand how the programming of the filters and chips work.  
  - [x] Change the chip label prefix "Technologies" to "Technology" 
  - [x] Change the chip label prefix "Tools" to "Tool"
  - [x] Using docker, test changes on both [current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js) and [current-projects-check.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects-check.js)
- [x] Release dependency on #4648 

### Resources/Instructions

-  Chip labels can be found on the [Projects page](http://hackforla.org/projects)
- [current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)
- [current-projects-check.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects-check.js)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)
- See associated ER for this issue: #4563

### PR DESCRIPTION
**Pull request by irais-valenzuela**
Fixes #4647

### What changes did you make?
  - In website/assets/js/current-projects.js I added the line 763 to check if the variable filterName was either "technologies" or "tools" and assign singularFormOfFilterName the singular form of filterName ("technology" and "tool"). 
  - In website/assets/js/current-projects.js I replaced filterName variable on line 770 with singularFormOfFilterName, to show singular form of "technologies" and "tools" text on the span tag.

### Why did you make the changes (we will use this info to test)?
  - Needed to ensure singular form of "technologies" and "tools" for filterName on chip labels to avoid confusion for users.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Before changing chip labels to singular

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
After changing chip labels to singular

</details>

### CODE REVIEW

File: assets/js/current-projects.js [hunk 1]
```diff
@@ -759,12 +759,15 @@ function dropDownFilterComponent(categoryName,filterArray,filterTitle){
 */
 
 function filterTagComponent(filterName,filterValue){
+    let copyOfFilterName = filterName
```
Comment (freaky4wrld) [inline assets/js/current-projects.js]: Using optional chaining here as well reduces the redundancy on the variable name `copyOfFilterName`. Just suggesting this piece of code 

```javascript

let copyOfFilterName = filterName === "tools" ? "tool" : filterName === "technologies" ? "technology" : filterName;

```

Comment (irais-valenzuela) [reply assets/js/current-projects.js]: Hi @freaky4wrld, thanks for taking a look. You make a good point. I made the changes.

**Review by freaky4wrld — APPROVED** [review]
@irais-valenzuela thanks for taking up the issue, the branches are correct and so does the issue linked.... the changes done rightly justify the requirement in the issue..... also the visuals provided are helpful. Manually testing shows correct and desired results, and you also pointed out the left out label in the issue, maybe that's covered in different issue. 
@roslynwythe can definitely provide insight on the issue.

PR approved!!

**Comment by t-will-gillis** [comment]
availability: 2/11/24
eta: eod 2/11/24

**Comment by t-will-gillis** [comment]
Apologies, have to push the eta back to 2/13

**Comment by t-will-gillis** [comment]
Hey @irais-valenzuela Great work on the issue and good communication with  @freaky4wrld. I opened this in Docker and can see that the "Technologies/Tools" filter defaults to "Technology" on the "Projects" page and to "Technology" and "Tools" on the "Projects-Check" page. (Minor note: in the comments for the screenshots, there is some extraneous text when the image is opened.)

I did a quick search to see if we have an issue to fix "Languages" and "Program Areas" and did not see one. This is outside the scope of your issue, but we should flag it with an ER. 

I also am noticing another small item that was not mentioned in the original issue, but that I think could be argued either way as to whether or not it is in the scope of this issue. When there are no results, the displayed message still uses the plural:
<details>
Screenshot 2024-02-13 203219
</details>  

If you would like to take a look and see if this is something you would like to address, that would be great, however, it will also be fine to add this to the ER. 

Thanks for working on this! Let me know if you would like to write the ER and whether you want to look at the search results or not. I will leave this issue un-merged for now.

**Review by t-will-gillis — APPROVED** [review]
Please see notes in comment

**Comment by irais-valenzuela** [comment]
@t-will-gillis Thanks for reviewing this! I just removed the extraneous text on the screenshots. I'd be happy to write an ER to address the "languages" and "program areas" issue. Is there any documentation you can point me to as far how to write an ER? In regards to the search results issue, i'd be happy to take that on. Would it be alright to address that issue within this same issue?

**Comment by t-will-gillis** [comment]
Hey @irais-valenzuela As I look through the original issue and back to the original ER, I believe that a new issue/ER needs to be inserted before the dependency can be released.  Since you completed what your issue asked for, I will merge this issue and speak with the rest of the team to make sure we are on track.

Great work again and thanks!

**Pull request merged**